src/EventStore.Client.Streams/WrongExpectedVersionResult.cs

```diff
@@ -1,17 +1,16 @@
-﻿using System.IO;
-
-#nullable enable
+﻿#nullable enable
 namespace EventStore.Client {
 	public readonly struct WrongExpectedVersionResult : IWriteResult {
 		public string StreamName { get; }
 		public long NextExpectedVersion { get; }
 		public long ActualVersion { get; }
 		public Position LogPosition { get; }
+		public StreamRevision NextExpectedStreamRevision { get; }
 
-		public WrongExpectedVersionResult(string streamName, long nextExpectedVersion, long actualVersion) {
+		public WrongExpectedVersionResult(string streamName, StreamRevision nextExpectedStreamRevision) {
 			StreamName = streamName;
-			ActualVersion = actualVersion;
-			NextExpectedVersion = nextExpectedVersion;
+			ActualVersion = NextExpectedVersion = nextExpectedStreamRevision.ToInt64();
```

Comment:
How come the actual version is the same as the next expected stream revision?

Reply:
You already know the expected version of the write you just attempted. I'm only leaving that property in there to not break the public api